### PR TITLE
Feature/add basic View,Controller and Model

### PR DESCRIPTION
**Pull request by ariririri**
add basic View,Controller and Model

This PR has two check points.
- questions at GUI is data from questions table
- can add question from view 

The above points are checked the following procedure.
1. `./build_run.sh`
2. access `localhost:8000/questions`
3. access `localhost:8000/questions/new`
4. add question and author, then push button
5. check the above question is added 

If there is no problem, please merge.

### CODE REVIEW

File: volume/samp_app/app/resources/questions/QuestionsController.jl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+function index()
+  """
+  :questions : questions resource(dir) 
+  :questiontop: name of viewfile, questiontop.jl.html
+  books = BillGatesBooks: @var books in jl.html
+  """
+  html!(:questions, Symbol("questiontop.jl.html"), questions = SearchLight.all(Question))
+  #html!(:questions, Symbol("questiontop.jl.html"), books = BillGatesBooks)
```
Comment (chaspy) [inline volume/samp_app/app/resources/questions/QuestionsController.jl]: why is it commnted out?

**Comment by chaspy** [comment]
should it be committed? `volume/samp_app/db/ask.sqlite`

**Comment by chaspy** [comment]
What is this? https://github.com/chaspy/ask/pull/15/commits/493b54e7f9625543e5b37dbe232ce63ea659c82a

File: volume/samp_app/app/resources/questions/QuestionsController.jl (hunk 1 above)
Comment (ariririri) [reply volume/samp_app/app/resources/questions/QuestionsController.jl]: This is the trial log. I remove it

**Comment by ariririri** [comment]
> should it be committed? `volume/samp_app/db/ask.sqlite`
If ask.sqlite does not exist, initial start neeed to add seed data.That's troublesome
I remove this when the system migrates postgres.

**Comment by chaspy** [comment]
title is same as #13 .
I changed this to distinguish each PR

**Comment by chaspy** [comment]
great!

**Pull request merged**